beta/src/pages/apis/usestate.md

```diff
@@ -4,38 +4,38 @@ title: useState
 
 <Intro>
 
-`useState` is a React Hook that lets you add a [state variable](/learn/state-a-components-memory) to your component.
+`useState` एक React Hook है जो आपको अपने कौम्पोनॅन्ट में एक [state वेरिएबल](/learn/state-a-components-memory) ऐड करने देता है|
 
 ```js
 const [state, setState] = useState(initialState)
 ```
 
 </Intro>
 
-- [Usage](#usage)
-  - [Adding state to a component](#adding-state-to-a-component)
-  - [Updating state based on the previous state](#updating-state-based-on-the-previous-state)
-  - [Updating objects and arrays in state](#updating-objects-and-arrays-in-state)
-  - [Avoiding recreating the initial state](#avoiding-recreating-the-initial-state)
-  - [Resetting state with a key](#resetting-state-with-a-key)
-  - [Storing information from previous renders](#storing-information-from-previous-renders)
-- [Reference](#reference)
+- [प्रयोग](#usage)
+  - [कौम्पोनॅन्ट में state ऐड करना](#adding-state-to-a-component)
+  - [पिचले state के आधार पर state अप्डेट करना](#updating-state-based-on-the-previous-state)
+  - [State में ऑब्जेक्ट्स और अर्रेस अपडेट करना](#updating-objects-and-arrays-in-state)
+  - [इनिशल state को फ़िर सृष्ट करने से बचना](#avoiding-recreating-the-initial-state)
+  - [ Key के सात state रिसेट करना](#resetting-state-with-a-key)
+  - [पिचले रेंडर के जानकारी को स्टोर करना](#storing-information-from-previous-renders)
+- [संदर्भ](#reference)
   - [`useState(initialState)`](#usestate)
   - [`set` functions, like `setSomething(nextState)`](#setstate)
-- [Troubleshooting](#troubleshooting)
-  - [I’ve updated the state, but logging gives me the old value](#ive-updated-the-state-but-logging-gives-me-the-old-value)
-  - [I've updated the state, but the screen doesn't update](#ive-updated-the-state-but-the-screen-doesnt-update)
-  - [I'm getting an error: "Too many re-renders"](#im-getting-an-error-too-many-re-renders)
-  - [My initializer or updater function runs twice](#my-initializer-or-updater-function-runs-twice)
+- [ट्रबलशूटिंग](#troubleshooting)
+  - [मैंने state का वैल्यू अपडेट किया है मगर log करते समय मुझे पुराना वैल्यू मिलता है](#ive-updated-the-state-but-logging-gives-me-the-old-value)
+  - [state अपडेट कर लिया है लेकिन स्क्रीन अपडेट नही हो रहा](#ive-updated-the-state-but-the-screen-doesnt-update)
+  - [मुझे एक एरर मिल रहा है: "Too many re-renders"](#im-getting-an-error-too-many-री-renders)
```

Comment:
`#im-getting-an-error-too-many-री-renders` => `#im-getting-an-error-too-many-re-renders`

lets not translate anything related to links or reflinks